.palantir/revapi.yml

```diff
@@ -1262,6 +1262,12 @@ acceptedBreaks:
         \ java.lang.Object[]) throws java.lang.Exception"
       justification: "Reduce visibilty of deprecated method"
     org.apache.iceberg:iceberg-core:
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.RewriteTablePathUtil.RewriteResult<T extends\
+        \ java.lang.Object>"
+      new: "class org.apache.iceberg.RewriteTablePathUtil.RewriteResult<T extends\
+        \ java.lang.Object>"
+      justification: "Adding an optional field"
```

Comment:
I think "Serialization across versions is not supported" is better and conform to result of `java.class.defaultSerializationChanged`